jobs_detector/main.py

```diff
@@ -26,7 +31,66 @@ def hacker_news(post_id, keywords, combinations):
     """
     # HINT: You will probably want to use the `BeautifulSoup` tool to
     # parse the HTML content of the website
-    pass
+
+    keywords = keywords.split(',')
+    combinations = combinations or [] # set this first so next line can iterate
+    combinations = [combinations.title() for combination in combinations]
+    main_page = requests.get(settings.BASE_URL.format(post_id))
+
+    # turn it into a BS4 object we can parse with html.
+    soupy = BeautifulSoup(main_page.text, 'html.parser')
+
+    # Find all the TR (table row) classes.  This will be the main thread, and each
+    # followup post/reply.
+    posting = soupy.find_all("tr", class_="athing")
+    top_posts = [p.get_text() for p in posting if p.find("img", width="0")]
```

Comment:
we found that looking for elements with class c00 was a better marker for job posts.